src/emu/x64emu_private.h

```diff
@@ -117,6 +117,7 @@ typedef struct x64emu_s {
     void*       init_stack; // initial stack (owned or not)
     uint32_t    size_stack; // stack size (owned or not)
     JUMPBUFF*   jmpbuf;
+    uintptr_t   old_savedsp;
```

Comment:
missing `#ifdef RV64` here too (because `old_savedsp` is irrelevant without `xSavedSP`

Reply:
tests are still not working with `BOX64_DYNAREC_CALLRET=1`, I need more debugging on that.